frontend/src/citizen-frontend/App.tsx

```diff
@@ -76,6 +77,11 @@ export default function App() {
                         component={requireAuth(MessagesPage, false)}
                       />
                     )}
+                    <Route
+                      exact
+                      path="/calendar"
+                      component={requireAuth(CalendarPage, false)}
+                    />
```

Comment:
Should this route be disabled when the feature is not enabled?

Reply:
A bit difficult to access user response here reliably, so I'll instead add `if(!user || !user.accessibleFeatures.reservations) return null` to the component itself